src/ng-select/items-list.ts

```diff
@@ -232,12 +232,12 @@ export class ItemsList {
     }
 
     private _stepToItem(steps: number) {
-        if (this._filteredItems.length === 0) {
+        if (this._filteredItems.length === 0 || this._filteredItems.every(x => x.disabled)) {
```

Comment:
This is not very efficient because you need to check every item on every step. What if you have 5000 items. Current solution only needed to check one item. Maybe you can break like this

```ts
if (this.markedItem.disabled && this._markedIndex === this._filteredItems.length - 1) {
   return;
}
```

Reply:
You can't break on the last one, since there could be item in the beginning which is not disabled. And `every` should break on first  not disabled item

Reply:
Agree, cool 👌